instrumentation/instrumentation.gradle

```diff
@@ -94,8 +94,29 @@ configurations {
   runtime.exclude group: 'io.opentelemetry', module: 'opentelemetry-api'
 }
 
+task shadowJarStep1(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar) {
```

Comment:
Comment explaining the need for the extra step?

Reply:
added!